pkg/util/image_util.go

```diff
@@ -106,7 +105,22 @@ func cachedImage(opts *config.KanikoOptions, image string) (v1.Image, error) {
 	if err != nil {
 		return nil, err
 	}
-	cacheKey := ref.Name()
-	fmt.Printf("CACHEKEY=%s", cacheKey)
-	return cache.LocalDestination(opts, cacheKey)
+
+	k8sc, err := k8schain.NewNoClient()
```

Comment:
I think there's an easier way to do this with no auth: https://github.com/google/go-containerregistry/blob/d54baf9aa28edb9b985a6b35b57e26e3410c2443/pkg/authn/anon.go